pkg/devfile/adapters/common/utils.go

```diff
@@ -32,7 +32,7 @@ const (
 
 	// Default Image that will be used containing the supervisord binary and assembly scripts
 	// use GetBootstrapperImage() function instead of this variable
-	defaultBootstrapperImage = "registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:1.1.3"
+	defaultBootstrapperImage = "registry.redhat.io/ocp-tools-4/odo-init-container-rhel8:1.1.4"
```

Comment:
Curious to know how authentication will be handled during the image pull ?

Reply:
Travis failure
``` 
✗  Pulling image registry.redhat.io/ocp-tools-4/odo-init-container-rhel8:1.1.4 [664ms]
```

This what i was expecting, but 4.2 and 4.5 confused me by passing the same test 😕

Reply:
Travis failure
```
[odo]  ✗  Waiting for component to start [4m] [WARNING x5: Failed]

```

Reply:
/hold